docker-compose.jsonnet

```diff
@@ -53,21 +56,27 @@
   // You can exec into the container and telnet to this port to get a python shell.
   backdoor_port:: 1234,
 
-  // Other nodes to backfill from. You should not include the local node.
+  // Other nodes to always backfill from. You should not include the local node.
```

Comment:
Probably add something like "If you're using the database to find peers, you should leave this empty"